DqtNotifications/terraform/app.tf

```diff
@@ -0,0 +1,153 @@
+locals {
+  auth_rule_name = "SendAndListenSharedAccessKey"
+  sku            = var.enable_private_endpoint == true ? "Premium" : var.sku
+  capacity       = local.sku == "Premium" && var.capacity <= 0 ? 1 : var.capacity
+       }
+
+# App Sercie Plan
+resource "azurerm_app_service_plan" "app_service_plan" {
+  name                = local.app_service_plan_name
+  location            = data.azurerm_resource_group.rgsb.location
+  resource_group_name = data.azurerm_resource_group.rgsb.name
+  kind                = "Linux"
+  reserved            = true
+
+  sku {
+    tier = "Dynamic"
+    size = "Y1"
+  }
+  
+
+  lifecycle {
+    ignore_changes = [
+      tags
+    ]
+  }
+}
+
+
+# Function App
+resource "azurerm_function_app" "functionAp" {
+  name                       = local.azurerm_function_app_name
+  location                   = data.azurerm_resource_group.rgsb.location
+  resource_group_name        = data.azurerm_resource_group.rgsb.name
+  app_service_plan_id        = azurerm_app_service_plan.app_service_plan.id
+  storage_account_name       = azurerm_storage_account.st.name
+  storage_account_access_key = azurerm_storage_account.st.primary_access_key
+
+  lifecycle {
+    ignore_changes = [
+      tags
+    ]
+  }
+}
+
+# Postgress Server
+resource "azurerm_postgresql_flexible_server" "postgres-server" {
```

Comment:
We need MSSQL here instead of Postgres. For now, we don't need it in the production environment either (we'll be borrowing an existing DB for the foreseeable future.)